go/streamlog/streamlog.go

```diff
@@ -58,6 +49,34 @@ var (
 		[]string{"Log", "Subscriber"})
 )
 
+var (
+	RedactDebugUIQueries = false
+	QueryLogFormat       = "text"
+	QueryLogFilterTag    = ""
+	QueryLogRowThreshold = uint64(0)
```

Comment:
for everything but `QueryLogFormat`, these are the zero values, so you can just do:

```go
var (
    RedactDebugUIQueries bool
    QueryLogFormat = "text"
    QueryLogFilterTag string
    QueryLogRowThreshold uint64
)
```

Reply:
can we un-export these as well? if it's helpful, what i have on [another PR](https://github.com/vitessio/vitess/pull/11122/files#diff-2f4a6e06176fc55981e641da8319d0722a1ae12ee347153a866fed8508ac5c7fR69) is:

1. `MyVariable` => `myVariable`
2. `func MyVariable() T { return myVariable }`